ui/src/components/Log/index.jsx

```diff
@@ -19,89 +19,105 @@ const LINE_NUMBER_MATCH = /L(\d+)-?(\d+)?/;
 const FOLLOW_STORAGE_KEY = 'follow-log';
 
 @withRouter
-@withStyles(theme => ({
-  '@global': {
-    'div.react-lazylog': {
-      backgroundColor: theme.palette.background.default,
-      fontFamily: 'Consolas, Monaco, Andale Mono, Ubuntu Mono, monospace',
-      fontSize: 13,
-      paddingTop: 4,
-      paddingBottom: theme.spacing.unit,
-      color: theme.palette.text.primary,
-      '-webkit-font-smoothing': 'auto',
+@withStyles(theme => {
+  const filterStyles = {
+    padding: '4px 8px',
+    lineHeight: 1,
+    '&.active': {
+      ...theme.mixins.successIcon,
+      borderRadius: '4px',
```

Comment:
The border radius could be set for all times, not just when the button is active. Also, we can omit the px and remove the quotations: `borderRadius: 4,`.

Reply:
Done.